StyleCop.Analyzers/StyleCopTester/Program.cs

```diff
@@ -24,6 +25,35 @@ internal static class Program
     {
         private static void Main(string[] args)
         {
+            CancellationTokenSource cts = new CancellationTokenSource();
+            Console.CancelKeyPress +=
+                (sender, e) =>
+                {
+                    e.Cancel = true;
+                    cts.Cancel();
+                };
+
+            SynchronizationContext previousContext = SynchronizationContext.Current;
+            try
+            {
+                var context = new DispatcherSynchronizationContext();
```

Comment:
Do we really want to take a dependency on wpf? With monodevelop supporting roslyn in the future we probably want to be able to run everything on mono too.